truncate_client/src/regions/active_game.rs

```diff
@@ -198,8 +200,24 @@ impl ActiveGame {
                             ui.add_space(10.0);
                         }
 
-                        ui.add_space(10.0);
+                        ui.add_space(5.0);
                     }
+
+                    if self.ctx.is_mobile {
```

Comment:
Can this be rendered inside `render_sidebar`? I'm assuming it's showing up in more or less the same place on the screen, and we have a check for `self.ctx.is_mobile` there.

Reply:
Nah this shows up under the timers. A screenshot would have been helpful but you'll see after merge 🤷